packages/cssnano-preset-default/src/__tests__/css-declaration-sorter.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava';
+import preset from '..';
+import {processCSSWithPresetFactory} from '../../../../util/testHelpers';
+
+const {processCSS, passthroughCSS} = processCSSWithPresetFactory(preset);
+
+test(
+    'keep order of short- and longhand properties',
+    passthroughCSS,
+    'h1{animation-name:a;animation:b}'
+);
+
+test(
+    'keep order of short- and longhand properties and sort remaining',
+    processCSS,
+    'h1{animation-name:a;z-index:0;animation:b;color:0;}',
+    'h1{animation-name:a;animation:b;color:0;z-index:0}'
+);
```

Comment:
We need a lot more tests here.

Reply:
Could you provide a bit more descriptive feedback?

Reply:
Note that the module itself has tests too: https://github.com/Siilwyn/css-declaration-sorter/blob/master/tests/test.js

Reply:
We need a range of tests to ensure that the module functions as expected, and to detect any future regressions. I expect tests covering cases where declarations should be rearranged, cases where they shouldn't, and cases with declarations of mixed precedence.

Reply:
The first test covers "cases where they shouldn't", the second has mixed precedence and declarations that should be rearranged right?

Reply:
We're going to need tests in cssnano to ensure proper integration. I'm sorry if that wasn't clear.

Reply:
Sorry but I still don't get what you're after. Both tests cover the cases you mentioned earlier.

Reply:
I'm after more tests so we can have confidence in the feature. We need tests to capture its function and any edge cases that might lurk about. Two test cases aren't nearly enough for that.

Reply:
@Siilwyn when can you add more tests?

\cc @evilebottnawi

Reply:
Yep need more tests, example with border/padding/margin, also all tests from related issues

Reply:
Once all other comments and questions are resolved I'll add more tests. 👍